Changes batch 1 (#1)
deimos-translations/lisathetimeless-translation: translation/maps.txt

\et[046]Don't ask\#me those\#questions\#right now!<#>Не задавай\#мне сейчас\#вопросов!
\et[046]Just do\#something!<#>Просто сделай\#что-нибудь!
\et[046]He was a\#good friend...<#>Он был\#хорошим другом...
\et[046]Da-aa-aamn!<#>Чё-ё-ё-ёрт!

> \et[046]You suck!<#>Ты - отстой!

Author: А сосёшь помоему лучше было

Reviewer: Какая-то калька с английского. Может, Дебил/дятел/кретин/утырок (т.к. не спас).

Author: ```suggestion
\et[046]You suck!<#>Ты мудак!
```

\et[208]Come on! Rocks are\#good for digestion.<#>Ну же! Камни полезны\#для пищеварения.
\et[000]Okay, okay! You got it.<#>Ладно, ладно! Понял.
\et[000]I have problems with it.<#>У меня с ним проблемы.

> \et[208]Hey!..<#>Эй!..

Author: Я всегда делал это через многоточие и знак и я буду продолжать делать так
```suggestion
\et[208]Hey!..<#>Эй...!
```

\et[006]But not always!<#>Но не всегда!
\et[006]In fights,\#depending on\#what you wear\#or who are,<#>В битвах, в зависимости\#от того, что вы носите,\#или как вас знают,
\et[006]you'll be\#targeted\#more often!<#>вы можете чаще\#становится вражеской\#целью!
\et[006]So be sure to\#wear what suits\#you the best!<#>Так что лучше носите\#то, что вам лучше подходит\#в конкретный момент!
\et[006]-Stylin' Terry<#>- Стиляга Терри
\et[002]You can't go here.<#>Ты не можешь пройти дальше.
\et[00]What's here?<#>Что там?
\et[002]We're fixing an\#unstable toilet,\#and it's a very\#dangerous job.<#>Мы чиним нестабильный\#туалет, и это очень\#опасная работа.
\et[002]It might even explode\#within seconds.<#>Он может взорваться\#в считанные секунды.
\et[00]Alright, I'd better go.<#>Ладно, я лучше пойду.

> \et[009]Uuugh... What's\#goin' on here?..<#>Ууухх... Что тут\#происходит?..

Author: ```suggestion
\et[009]Uuugh... What's\#goin' on here?..<#>Ууухх... Что тут\#происходит...?
```

<!-- Map Name --><#>Canyon
<!-- In-game Displayed Name:  --><#>
\et[017]Finally someone's\#here!<#>Наконец-то, здесь\#кто-то есть!

> \et[017]Your money or your life\^<#>Кошелёк или жизнь!\^

Author: \^ это мгновенное прерывание текста, т. е. текстбокс закрывается, так что здесь не должно быть знака
```suggestion
\et[017]Your money or your life\^<#>Кошелёк или жизнь\^
```

\et[08]God, I love liquor.<#>Боже, я люблю пить.
\et[09]Ironton Town feels\#just like home.<#>Айронтон для меня\#словно дом.
\et[09]Just like that\#trailer park\#I used to\#live in...<#>Прямо как\#трейлерный\#парк, в котором\#я когда-то жил...
\et[09]Home sweet home.<#>Дом, милый дом.
\et[10]Peter Piper\#pecked a pick\#of pecker\#pickles...<#>На дворе трава,\#на траве дрова,\#не руби двора...
\et[10]A pick of Peter\#Piper, pickled\#pecker pick...<#>На дворе трава,\#на траве двора...
\et[10]Fuck,\| who\#comes up\#with this\#shit?!<#>Блять,\| кто\#этот бред\#придумывает?!
\et[10]Shelly shells\#sally sells\#on the she\#sore...<#>Клара у Карла\#украла коллары...
He's sleeping.<#>Он спит.
\et[20]This landscape does\#no favors for us\#cripples.<#>Этот ландшафт\#вредит нам,\#инвалидам.

> \et[20]Can we get some\#'accessibility' ramps\#around here?<#>Неужели нельзя тут\#поставить парочку\#пандусов?

Author: Точно

\et[08]Alright! Now that we've got\#your attention, listen up!<#>Так! Теперь, когда мы\#привлекли ваше внимание, слушайте!
\et[08]We're the TNT Team,<#>Мы - команда ТНТ,
\et[08]the most dangerous\#and destructive group this side of Olathe!<#>самая опасная и разрушительная\#группа в этой части Олейта!
\et[08]However, these days\#we're running low on\#good cannon fodder,<#>Хотя, у нас в последние\#дни недостаток пушечного\#мяса...

> \et[08]so we're offering you fellas\#the opportunity to join our ranks.<#>так что мы даём вам, ребята,\#возможность вступить в нашу группу.

Author: ```suggestion
\et[08]so we're offering you fellas\#the opportunity to join our ranks.<#>Так что мы даём вам, ребята,\#возможность вступить в нашу группу.
```

\et[05](Would you like it\#to save your game?)<#>(Вы хотите\#сохранить игру?)
Yes<#>Да
No<#>Нет

> \et[05]A blank wanted poster.<#>Пустой разыскной постер.

Author: ```suggestion
\et[05]A blank wanted poster.<#>Пустой разыскной плакат.
```

<!-- In-game Displayed Name:  --><#>
You got a Crab.<#>Вы получили краба.
\et[000]Oh my.<#>О боже.
You dropped your Diving helmet in fear.<#>Вы в ужасе опрокинули свой водолазный шлем.

> Flush the toilet<#>Спустить воду

Author: Ну смыть унитаз же просто круче звучит

Reviewer: Так ты словно унитаз в унитазе смываешь.

Author: > Так ты словно унитаз в унитазе смываешь.

Ну дак тут вроде контекст понятный

\et[024]Then I'm afraid I haven't.<#>Тогда, боюсь, не видел.
\et[024]All I know is fish.<#>Я знаю лишь рыб.
\et[024]All I've seen is fish.<#>Я видел лишь рыб.

> \et[024]And all I'll\#continue to do.<#>И буду видеть\#лишь рыб.

Author: Ну так как он следующей строкой как бы объясняет, что он хотел сказать здесь, предлагаю так
```suggestion
\et[024]And all I'll\#continue to do.<#>И я продолжу.
```

Reviewer: Мне понравился повтор "рыб", он звучит забавнее оригинала. И завершается "рыбачить". В любом случае, продолжить - глагол переходный, так что надо "И продолжу видеть."

Author: > Мне понравился повтор "рыб", он звучит забавнее оригинала. И завершается "рыбачить". В любом случае, продолжить - глагол переходный, так что надо "И продолжу видеть."

Ну тогда можно сделать:

`И я продолжу ловить...`
`Рыб.`

Author: ```suggestion
\et[024]And all I'll\#continue to do.<#>И я продолжу\#ловить...
```

> \et[024]Is fish.<#>Рыбачить.

Author: ```suggestion
\et[024]Is fish.<#>Рыб.
```

\et[002]Was he a fish?<#>Он рыба?
\et[00]No?<#>Нет?
\et[002]Then I'm afraid I haven't.<#>Тогда, боюсь, не видел.
\et[02]All I know is fish.<#>Я знаю лишь рыб.
\et[002]All I've seen is fish.<#>Я видел лишь рыб.

> \et[02]And all I'll continue to do.<#>И буду видеть лишь рыб.

Author: ```suggestion
\et[02]And all I'll continue to do.<#>И буду продолжать.
```

Author: ```suggestion
\et[02]And all I'll continue to do.<#>И я продолжу ловить...
```

> \et[002]Is fish.<#>Рыбачить.

Author: ```suggestion
\et[002]Is fish.<#>Рыб.
```

I don't have it, sorry. (Lie)<#>У меня его нет, прости. (Ложь)
\et[000]Uh, yeah.\#I found it.<#>А, да.\#Я нашёл его.
You have lost Gasoline Can<#>Вы потеряли канистру с бензином.

> \et[007]Good.\. Here's\#your end\#of the\#bargain.<#>Отлично.\.\#Вот награда,\#как договаривались.

Author: А что в прошлом варианте не так

Reviewer: Сейчас дошло, что не так, верно будет "Вот _твоя_ часть сделки."

Author: ```suggestion
\et[007]Good.\. Here's\#your end\#of the\#bargain.<#>Отлично.\.\#Вот твоя часть\#сделки.
```
